### DISCUSSION TITLE
In view mode swipe left and right to show next and and previous note

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#188] [Feature Requests]
[Karym]: In view mode when you swipe left it shows the next note..  when you swipe right it shows the previous note..

**Comment by irmod** [comment]
@damionx7 

My experience is that swiping in View Mode is a bad idea... Swiping is useful for accessing the left and right Navigation Drawer...

The solution in the video is convenient for me

https://github.com/user-attachments/assets/927606c8-3c9c-4531-9dda-9fb4f441758b
